load/Validate.py

```diff
@@ -76,6 +77,33 @@ def process(self, filesets):
 		#duplicates = find.findDuplicates()
 		#find.moveDuplicates(duplicates)
 
+	def validateTextPlainFilesets(self, texts, inp, filePath):
```

Comment:
@bradflood I have created this new method to invoke validateFileset method for a text_plain fileset and to invoke new invokeBiblePublisher method.